testing/run_tests.sh

```diff
@@ -43,8 +76,14 @@ shard_tests=$(bazel query 'tests(//tensorflow_probability/...)' |
 #   --action_env -- specify environment vars to pass through to action
 #     environment. (We need these in order to run inside a virtualenv.)
 #     See https://github.com/bazelbuild/bazel/issues/6648 and b/121259040.
-echo "${shard_tests}" | xargs bazel test --copt=-O3 --copt=-march=native \
-  --test_tag_filters=-gpu,-requires-gpu-sm35 \
-  --test_timeout 300,450,1200,3600 --build_tests_only \
-  --action_env=PATH --action_env=LD_LIBRARY_PATH \
-  --test_output=errors
+echo "${shard_tests}" \
+  | xargs bazel test \
+    --copt=-O3 \
+    --copt=-march=native \
+    --notest_keep_going \
+    --test_tag_filters=-gpu,-requires-gpu-sm35 \
+    --test_timeout 300,450,1200,3600 \
+    --build_tests_only \
```

Comment:
I actually think this isn't needed, since we explicitly pass only tests to bazel test.

Reply:
ah good point!